gazelle/python/python_test.go

```diff
@@ -21,18 +21,15 @@ package python_test
 
 import (
 	"bytes"
-	"context"
 	"errors"
+	"github.com/bazelbuild/bazel-gazelle/testtools"
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+	"github.com/ghodss/yaml"
```

Comment:
nit: is this expected behaviour of `goimports`?

Reply:
oops